lib/config/configItems.joi.js

```diff
@@ -150,6 +151,10 @@ const qpKafkaJoi = kafkaJoi.append({
     replication: extensionKafkaJoi,
 });
 
+const supportedLifecycleRulesJoi =  joi.array().items(
+    joi.string().valid(...supportedLifecycleRules))
```

Comment:
Not sure we want to limit the valid choices to only what's in `supportedLifecycleRules`.

Reply:
if we support more, the new rules must be added to `supportedLifecycleRules`.

(maybe later we will need to have a different default, e.g. if some rules must not be enabled by default ; but for now we don't need this, so the whole list is also the default)